ArcGISRuntimeSDKQt_CppSamples/Maps/DisplayMap/arcgisruntime.pri

```diff
@@ -1,18 +1,20 @@
 #-------------------------------------------------
-#  Copyright 2017 ESRI
+# Copyright 2022 Esri.
 #
-#  All rights reserved under the copyright laws of the United States
-#  and applicable international laws, treaties, and conventions.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# http://www.apache.org/licenses/LICENSE-2.0
 #
-#  You may freely redistribute and use this sample code, with or
-#  without modification, provided you include the original copyright
-#  notice and use restrictions.
-#
-#  See the Sample code usage restrictions document for further information.
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
 #-------------------------------------------------
 
 contains(QMAKE_HOST.os, Windows):{
-  iniPath = $$(ALLUSERSPROFILE)\\EsriRuntimeQt\\ArcGIS Runtime SDK for Qt $${ARCGIS_RUNTIME_VERSION}.ini
+  iniPath = $$(ALLUSERSPROFILE)\EsriRuntimeQt\ArcGIS Runtime SDK for Qt $${ARCGIS_RUNTIME_VERSION}.ini
```

Comment:
I would assume we should keep the double backslash version, but if this works I'm find to standardize on this.

Reply:
Both work it seems, so I'm not sure which we should use. The majority of samples use single slashes, but double slashes seems safer.

Reply:
Let's go with double backslash then. That's what we have the SDK templates.